src/analytics.ts

```diff
@@ -92,7 +92,7 @@ const withAnalytics = ({
     }
 };
 
-export type TrackingEvent = Readonly<{
+type LegacyAnalyticsEvent = {
```

Comment:
mark as deprecated?

Reply:
This isn't exported, so there isn't a big reason to deprecate the type. Also, novum will still use them for some time